doc/classes/Node3D.xml

```diff
@@ -263,7 +265,7 @@
 		<method name="update_gizmos">
 			<return type="void" />
 			<description>
-				Updates all the [Node3DGizmo]s attached to this node.
+				Updates all the Node3D gizmos attached to this node.
```

Comment:
```suggestion
				Updates all the [Node3D] gizmos attached to this node.
```

We don't have good style rules for this, but given that other usages are reference-formatted, I'd keep it consistent.

Reply:
Fixed, together with a few other occurrences (+ grammar errors on those lines).